### ISSUE TITLE
Exclusive import

### ISSUE THREAD

**Issue by AllanKirk** [KevinJump/jumps.umbraco.usync#64]
Here is an idea I would like, as an option: Exclusive import.

We are working with branches, and we need a way to clear data from the database, if it is not part of the current branch. The exclusive import would be a complete import, but it would also delete things that are not on disk.

Usecase:
1. I work on feature A, where I am doing some major restructuring in my document-types. This works just fine, as long as I stay on branch A.
2. My colleague asks me to help with something on feature B on branch B, so I switch branch.
3. Because the import will only import document-types on branch B, all my changes from Branch A will still be in the database.
4. I would press "Exclusive import", and uSync would basically delete everything from the database, and import everything from disk.

Right now my workaround is to delete the entire database, add a clean database from a script I have, and do an import using uSync.

**Comment by KevinJump** [comment]
yes - this is a good idea - moved to https://github.com/KevinJump/uSync/issues/81
